README.md

```diff
@@ -142,6 +142,19 @@ Then run either `node ./server.js` from the api directory or `grunt dev-api` fro
 
 Navigate your browser to [`http://localhost:5988/medic/login`](http://localhost:5988/medic/login).
 
+### Working on local environment with real devices. 
+
+This process is relevant for viewing the Medic webapp on mobile devices when the api service is run on a developer machine running Webapp >v3.5.0. Webapp v3.5.0 relies on service workers, which require a valid HTTPS certificate to function. Follow these steps to make your developer build accessible from your android device at the trusted url created by ngrok.
+
+1. Create an ngrok account at https://ngrok.com/ 
+2. Follow instructions on downloading and linking your computer to your ngrok account.
+3. Start the webapp. This can be via docker, grunt, debug, horti, etc....
```

Comment:
I think `debug` isn't an actual option?

Reply:
I was thinking that if I set vscode to run api in debug I could use ngrok to hit it. I can remove that if you feel it's still not valid.

Reply:
I'd recommend either removing it or adding a section to explain it. Whichever you feel is better?

Reply:
I went ahead and removed it. If it comes up as a question we can add it back later.